### PR TITLE
Add config to override JDK default compiler

### PR DESCRIPTION
**Pull request by skoczko**
This only works for running tests and when running via `mvn exec:exec`
This DOES NOT fix the issue when running from the container -- for that
we will need to alter the Java startup script in the base image.

### CODE REVIEW

**Comment by skoczko** [comment]
@adamdickmeiss Yikes, I see that you already have a branch for this. I'lll close it then and add the exec config on top of your branch.

File: etc/launch-from-modules.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+cd ../server/target/
```
Comment (adamdickmeiss) [inline etc/launch-from-modules.sh:2]: Can you put it in server instead? I don't like so many directories in root.

File: server/pom.xml [hunk 2]
```diff
@@ -164,13 +164,19 @@
...
-                    <Main-Class>io.vertx.core.Launcher</Main-Class>
-                    <Main-Verticle>org.folio.metastorage.server.MainVerticle</Main-Verticle>
+                    <Main-Class>${vertx.launcher}</Main-Class>
+                    <Main-Verticle>${vertx.verticle}</Main-Verticle>
                     <Multi-Release>true</Multi-Release>
                   </manifestEntries>
                 </transformer>
               </transformers>
-              <artifactSet />
+              <artifactSet>
```
Comment (adamdickmeiss) [inline server/pom.xml:173]: I'd prefer we do not exclude these . Because it will make the fat jar without options fail upon start.

**Review by adamdickmeiss — CHANGES_REQUESTED** [review]

**Review by adamdickmeiss — APPROVED** [review]
descriptors/DeploymentDescriptor-template.json should be updated too. It will break as exclusions are removed. The change can be done  in separate PR.

**Pull request merged**